tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/Microsoft.DotNet.SourceBuild.Tasks.csproj

```diff
@@ -4,6 +4,15 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="../../../src/core-setup/src/managed/Microsoft.DotNet.PlatformAbstractions/Platform.cs" />
+    <Compile Include="../../../src/core-setup/src/managed/Microsoft.DotNet.PlatformAbstractions/RuntimeEnvironment.cs" />
```

Comment:
Can you define a property for the path `../../../src/core-setup/src/managed/Microsoft.DotNet.PlatformAbstractions`?  I think it would make this easier to read.